tests/test_batch.py

```diff
@@ -0,0 +1,71 @@
+from unittest.mock import Mock, MagicMock, patch
+
+import pytest
+
+from sdk import SDK
+from sdk.batch import Batch
+
+
+class TestBatch:
+    @pytest.fixture(autouse=True)
+    @patch("sdk.Client")
```

Comment:
Discussed offline: instead of mocking whole Client, mock only the `requests` part. Create fixtures folder to gather in the same place all responses of the API.

Maybe we could setup a script to automatically update the fixtures, not sure.

Reply:
Done